libobs/util/deque.h

```diff
@@ -294,6 +294,19 @@ static inline void deque_pop_front(struct deque *dq, void *data, size_t size)
 		dq->start_pos -= dq->capacity;
 }
 
+static inline void deque_free_front(struct deque *dq, size_t size)
```

Comment:
This isn't really necessary, you can just use `deque_pop_front()` with `void* data` being `NULL`.

Reply:
Changed, `deque_free_front` removed.